### PR TITLE
[RFR] Improve look & use ark

### PR DESCRIPTION
**Pull request by touv**

### CODE REVIEW

File: src/app/js/formats/istex/fetchIstexData.js [hunk 1]
```diff
@@ -18,12 +20,23 @@ export const parseFetchResult = fetchResult => {
         throw new Error(fetchResult.error);
     }
     const { response: { total, hits } } = fetchResult;
-
+    const { protocol, host } = URL.parse(istexApiUrl);
+    console.log('>>>>>>>>>>>>>>>>>>>>>>>', hits);
```
Comment (ThieryMichel) [inline src/app/js/formats/istex/fetchIstexData.js]: debug spotted

File: src/app/js/formats/istex/fetchIstexData.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import fetch from '../../lib/fetch';
 import composeAsync from '../../../../common/lib/composeAsync';
+import URL from 'url';
 
 const istexApiUrl = process.env.ISTEX_API_URL;
 
 export const getUrl = ({ props: { resource, field }, page, perPage }) => {
     const value = resource[field.name];
-    const output = 'id,title,publicationDate,fulltext,abstract';
+    const output =
```
Comment (ThieryMichel) [inline src/app/js/formats/istex/fetchIstexData.js:9]: nice to have: this could be refactored to work like injectData and allow ssr for this format.

**Review by ThieryMichel — CHANGES_REQUESTED** [review]

File: src/app/js/formats/istex/fetchIstexData.js (hunk 2 above)
Comment (touv) [reply src/app/js/formats/istex/fetchIstexData.js:9]: you're right, but I don't know how to do that, and I have no time to explore this new way

File: src/app/js/formats/istex/fetchIstexData.js (hunk 1 above)
Comment (touv) [reply src/app/js/formats/istex/fetchIstexData.js]: oups

**Review by ThieryMichel — APPROVED** [review]

**Pull request merged**